targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CPROTO_064_SB/device/COMPONENT_CM4/TOOLCHAIN_GCC_ARM/cyb06xx7_cm4_dual.ld

```diff
@@ -20,6 +20,7 @@
 ********************************************************************************
```

Comment:
Commit message for "GCC linker changes for CY8CPROTO_064_SB", says "cymcuel tool changes to notify about the new flash region length.". I don't see any cymcuel tool changes in this commit.

If you are referring to the `__cy_memory_0_length   = 0x00100000;` change, then say something along the lines of "correctly specify flash size as 1 MiB, not 832 KiB"

Reply:
Fixed.